packages/gui/index.ts

```diff
@@ -7,6 +7,7 @@ export * from './components/DrawerContainer';
 export * from './components/Hoverable';
 export * from './components/Input';
 export * from './components/ModalManager';
+export * from './components/Passcode';
```

Comment:
I don't think we want to put `Passcode` in `@walless/gui`, this package is only used for general UI components, right? `Passcode` seems like only belonged to `apps/wallet`.

Need a confirmation from @cloudle

Reply:
I think we can put `passcode` here, just a single component with its logic, without any understanding about feature